CMakeLists.txt

```diff
@@ -70,13 +70,14 @@ string(TOUPPER ${META_PROJECT_ID} META_PROJECT_ID)
 # 
 
 # Project options
-option(BUILD_SHARED_LIBS      "Build shared instead of static libraries."              ON)
-option(OPTION_SELF_CONTAINED  "Create a self-contained install with all dependencies." OFF)
-option(OPTION_BUILD_TESTS     "Build tests."                                           ON)
-option(OPTION_BUILD_GPU_TESTS "Build tests that require an OpenGL context."            ON)
-option(OPTION_BUILD_DOCS      "Build documentation."                                   OFF)
-option(OPTION_BUILD_TOOLS     "Build tools."                                           ON)
-option(OPTION_BUILD_EXAMPLES  "Build examples."                                        OFF)
+option(BUILD_SHARED_LIBS              "Build shared instead of static libraries."              ON)
+option(OPTION_SELF_CONTAINED          "Create a self-contained install with all dependencies." OFF)
+option(OPTION_BUILD_TESTS             "Build tests."                                           ON)
+option(OPTION_BUILD_GPU_TESTS         "Build tests that require an OpenGL context."            ON)
+option(OPTION_BUILD_DOCS              "Build documentation."                                   OFF)
+option(OPTION_BUILD_TOOLS             "Build tools."                                           ON)
+option(OPTION_BUILD_EXAMPLES          "Build examples."                                        OFF)
+option(OPTION_BUILD_WITH_BOOST_THREAD "Use std::regex instead of boost"                        OFF)

```

Comment:
First, did you mean to say `thread` instead of `regex`? Second, did you mean to say the reverse order? Third, should this end with a period? It looks like it should be: `Use boost::thread instead of std::thread.`